src/main/kotlin/org/jetbrains/teamcity/rest/api.kt

```diff
@@ -181,3 +193,20 @@ interface TriggeredInfo {
     val user: User?
     val build: Build?
 }
+
+interface Trigger{
```

Comment:
Please use 'Complete Statement' action when writing code to ensure that it follows the project's code style (e.g. insert space before {.